deeppavlov/models/ranking/siamese_model.py

```diff
@@ -31,7 +31,7 @@ class SiameseModel(NNModel):
 
     def __init__(self,
                  batch_size: int,
-                 num_context_turns: int = 1,
+                 num_context_turns: int,
```

Comment:
Why do you remove default value?

Reply:
I think I can revert it back, just need some testing...

Reply:
Reverted, breakes nothing. https://github.com/deepmipt/DeepPavlov/blob/8c9e9fd176abbb799b075512d10b1d7effa2ed17/deeppavlov/models/ranking/siamese_model.py#L34

Reply:
Thank you.